### PR TITLE
build scripts

### PR DESCRIPTION
**Pull request by progers-tech**

### CODE REVIEW

File: pipenv.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cd infrastructure
+terraform init -backend-config "bucket=$1-terraform-state"
+terraform workspace new $2 || terraform workspace select $2
+terraform apply -auto-approve -input=true
+
+sleep 5
```
Comment (blackmamo) [inline pipenv.sh:6]: You are a snorlax!

**Review by blackmamo — DISMISSED** [review]
Would like to see the sleep removed if possible

File: pipenv.sh (hunk 1 above)
Comment (progers-tech) [reply pipenv.sh:6]: :zzz:

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ $# -ne 2 ]
+then
+    echo "Syntax: build.sh <app_name> <tf_workspace>"
```
Comment (blackmamo) [inline setup.sh]: This is now wrong

**Review by blackmamo — CHANGES_REQUESTED** [review]

**Review by blackmamo — APPROVED** [review]

**Pull request merged**